### PR TITLE
Hides the number if the like count is less than 3

### PR DESCRIPTION
**Pull request by javierarce**
What is says on the tin :point_up: 

Original ticket: https://github.com/CartoDB/cartodb/issues/4007

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4338/))

**Comment by javierarce** [comment]
@xavijam: a little PR :gift: for you, can you have a look, please?

File: lib/assets/javascripts/cartodb/common/views/likes/template.jst.ejs [hunk 1]
```diff
@@ -1 +1 @@
-<i class="iconFont iconFont-Heart--fill LikesIndicator-icon"></i><%= m.get('likes') %>
+<i class="iconFont iconFont-Heart--fill LikesIndicator-icon"></i><% if (m.get('likes') > 2) { %><span class="LikesIndicator-count"><%= m.get('likes') %></span><% } %>
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/views/likes/template.jst.ejs]: try to pass a variable here, like: ```likes```. and not the whole model.

File: lib/assets/javascripts/cartodb/public/public_like.js [hunk 2]
```diff
@@ -45,11 +45,25 @@ cdb.open.LikeView = cdb.core.View.extend({
...
+  
+  },
+
   _unhighlightHeart: function() {
 
     var $button  = $(".js-like");
     var $icon    = $(".js-like .icon");
-    var $counter = $(".js-like .counter");
+
+    this._updateCounter();
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/public/public_like.js]: Why do we need to update the counter within ```_unhighlightHeart``` or ```_highlightHeart```?
I've seen that _updateCounter is called after likes is changed.

Comment (javierarce) [inline lib/assets/javascripts/cartodb/public/public_like.js]: You're totally right, removed.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4345/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4348/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4350/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4351/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4352/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4354/))

**Comment by javierarce** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4366/))

**Pull request merged**